tests/testthat/test-observe.R

```diff
@@ -92,7 +92,7 @@ test_that("observe messages/warns/errors informatively", {
 
 test_that("observe() works with either specify() interface", {
   # unnamed formula argument
-  expect_equal(
+  expect_equivalent(
```

Comment:
Why this change?

Reply:
`infer` objects now bring along a `formula` if it's given to drop directly to `fit` and otherwise parses it on-the-fly from `response` and `explanatory`. Since `expect_equal` looks at attributes, it flags that `attr(x, "formula")` exists in one and not the other. The test that follows this one up tests that other attributes are brought along the same across `observe` vs full infer pipeline. :-)